daft/daft.pyi

```diff
@@ -1025,6 +1025,9 @@ class PyExpr:
     def log(self, base: float) -> PyExpr: ...
     def ln(self) -> PyExpr: ...
     def exp(self) -> PyExpr: ...
+    def bitwise_and(self, other: PyExpr) -> PyExpr: ...
+    def bitwise_or(self, other: PyExpr) -> PyExpr: ...
+    def bitwise_xor(self, other: PyExpr) -> PyExpr: ...
```

Comment:
Since these are equivalent to `__and__`, `__or__`, and `__xor__`, could you just use those instead of defining additional functions? Alternatively, maybe we should have a code path for these where they must be numeric values and not boolean

Reply:
Yeah you're right, they are equivalent. Don't think we need a completely separate code path for now though.